safe/impact_function/impact_function.py

```diff
@@ -252,6 +252,7 @@ def __init__(self):
         self._name = None  # e.g. Flood Raster on Building Polygon
         self._title = None  # be affected
         self._unique_name = None  # EXP + On + Haz + DDMMMMYYYY + HHhMM.SS.ms
+        self._multi_exposure_name = None
```

Comment:
What is the good reason to have this name here?
impact_function.py is not aware of the multiexposure. That's why we have the multi_exposure_wrapper to manage all impact_function individually.
I would prefer very much if we can remove this. If it's for the provenance, can't you write it in the wrapper? I really tried to not touch the impact_function.

Reply:
ah this is my old approach, for provenance. I was trying to find a way to identify whether an IF is a member of Multi-exposure IF, but I ended up using the group registry in the legend to identify that case. we can safely remove this, thanks anyway.